backend/internal/storage/postgres/users.go

```diff
@@ -8,22 +8,45 @@ import (
 
 func (db *DB) CreateUser(ctx context.Context, userToCreate models.User) (models.User, error) {
```

Comment:
add createdAt field where applicable

Reply:
would you like me to add "created at" to our schema? Currently the user table doesn't have a created at field

Reply:
same as above